docs/quickwit-cli/README.md

```diff
@@ -0,0 +1,193 @@
+# Quickwit CLI
+
+Quickwit CLI is a tool that makes it easy to index and search structured or unstructured data from the command line. It consumes datasets consisting of newline-delimited JSON objects with arbitrary keys and produces indexes that can be stored locally or remotely on an object storage such as Amazon S3 and queried with subsecond latency.
+
+Key features include the following:
+- schemaless
+- local and remote indexes
+- native support for time-series datasets
```

Comment:
I think people imagine something different when they hear time-series datasets. Maybe "native support for timestamps" (though I agree it sounds dumb)

Reply:
It's more related to time partitionning or sharding
maybe put "native support for time sharding"